parts/k8s/kubernetesprovisionsource.sh

```diff
@@ -55,10 +55,12 @@ wait_for_file() {
 }
 apt_get_update() {
     retries=10
+    apt_update_output=/tmp/apt-get-update.out
     for i in $(seq 1 $retries); do
         timeout 30 dpkg --configure -a
-        timeout 120 apt-get update 2>&1 | grep -x "[WE]:.*"
-        [ $? -ne 0  ] && break || \
+        timeout 120 apt-get update 2>&1 | tee $apt_update_output | grep -E "^([WE]:.*)|([eE]rr.*)$"
```

Comment:
engage output file, and adding additional regular expression to match an error output line that begins with "err" or "Err"